marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java

```diff
@@ -154,7 +148,7 @@ public MarkLogicRepository(String host, int port, String user, String password,
     /**
      * constructor inited with java api client DatabaseClient
      *
-     * @param databaseClient
+     * @param databaseClient the java api client to be used
```

Comment:
Here's the corresponding javadoc from jena:

```
/**
     * Constructor.
     * 
     * @param client
     *            a Java Client API DatabaseClient. Can be made with
     *            com.marklogic.client.DatabaseClientFactory
     */
```

If you include packages it's easier for somebody to learn what's going on with this library